operator/api/v1alpha1/observatorium_types.go

```diff
@@ -178,11 +178,35 @@ type APITenant struct {
 	OIDC TenantOIDC `json:"oidc"`
 }
 
+type TLSSecret struct {
+	CertFile       string `json:"certFile"`
+	Name           string `json:"name"`
+	PrivateKeyFile string `json:"privateKeyFile"`
+	ReloadInterval string `json:"reloadInterval"`
```

Comment:
We will need an optional field to load the CA that signed the certificate. This is optional because the healthchecker needs to know that the certificate is valid and if the CA is not in the system certs then it needs to be specified. Note, this is different from an mTLS CA used to check clients of the API; this is for clients to validate the server's cert